src/index.js

```diff
@@ -0,0 +1,59 @@
+import * as task from './live.js';
+import './style.css';
+
+let list = [
+  { description: 'Set up a new project with webpack', isCompleted: false, index: 0 },
+  { description: 'Set up a new project with bundle webpack', isCompleted: false, index: 1 },
+  { description: 'Create an index.js file', isCompleted: false, index: 2 },
+  { description: 'Write a function to iterate over the tasks array and populate an HTML', isCompleted: false, index: 3 },
+];
+
+function todoList() {
+  if (window.localStorage.getItem('localTasks')) {
+    const localTasks = window.localStorage.getItem('localTasks');
+    list = JSON.parse(localTasks);
+  }
+  document.querySelector('.todo-list').innerHTML = '';
+  list.forEach((item) => {
```

Comment:
Kindly 🙏🏾  practice the use of array destructuring while using objects as arguments to functions. ♻️